conf/application.conf

```diff
@@ -214,3 +214,7 @@ accessibility-urls {
 }
 
 host = "http://localhost:9582"
+
+toggle {
+  samples = true
+}
```

Comment:
Could we review the name of this toggle? 

Should it be called `samplesNotAccepted` or `samplesRejected` or something like that? I think at the moment it's a bit hard to figure out what will happen if this toggle is turned on or off as the default is `true` which seems like it should mean that samples are allowed

Reply:
yeah sounds we have changed it to samplesNotAccepted now

Reply:
Awesome thanks @gorkemunsal

Reply:
I've merged those config PRs after the toggle name change